jdbc-examples/src/main/java/io/vertx/example/jdbc/query_params/JDBCExample.java

```diff
@@ -26,8 +27,7 @@ public void start() {
 
     vertx.executeBlocking(future -> {
       try {
-        stopDockerDatabase(true);
-        startDockerPostgres();
+        startDocker(POSTGRESQL);
```

Comment:
Does this work? I see you deleted the file `DockerDatabase.java` - where is the method `stopDockerDatabase()` used on line 83 implemented?

Reply:
Ah ok nevermind, it was just moved. :)